index.html

```diff
@@ -31,21 +30,17 @@
             <div class="hamburger-icon">
                 <img src="images/icon/menu-icon.png" class="menu-icon" alt="menu icon">
             </div>
-            <div class="nav-list">
+            <div class="desktop">
                 <ul class="nav-items">
-                    <li>
-                        <a href="#" class="nav-item">Portfolio</a>
-                    </li>
-                    <li>
-                        <a href="#" class="nav-item">About</a>
-                    </li>
-                    <li>
-                        <a href="#" class="nav-item">Contact</a>
+                    <li><a href="#" class="nav-item">Portfolio</a></li>
+                    <li><a href="#" class="nav-item">About</a></li>
+                    <li><a href="#" class="nav-item">Contact</a></li>
+                    <li class="nav-item">
```

Comment:
- You did a great job making a responsive design. 🎉  How about adding more paddings for each list item so that it matches the Figma design? Can you please fix this? 🥇 

Figma Design
<img width="536" alt="Screenshot 2022-12-10 at 10 31 48 PM" src="https://user-images.githubusercontent.com/24832846/206866504-90b23976-889d-4a99-ad2d-19e5db6f0f8a.png">

Your design
<img width="501" alt="Screenshot 2022-12-10 at 10 29 39 PM" src="https://user-images.githubusercontent.com/24832846/206866424-382fee43-1109-4b66-82d5-2b91ae87eba9.png">